src/modules/endpoint/DatabaseConnector.js

```diff
@@ -135,7 +146,10 @@ class DatabaseConnector {
     return `${CACHE_DIR_PATH}/cities.json`
   }
 
-  async storeLastUpdate (lastUpdate: Moment, context: DatabaseContext) {
+  async storeLastUpdate (lastUpdate: Moment | null, context: DatabaseContext) {
+    if (lastUpdate === null) {
+      return
+    }
```

Comment:
Because of the flow types in the `DefaultDataContainer`. `loadLastUpdate` returns `null` if there it has not been set yet. Because of this the `Cache` has to have the type `Moment | null` and hence the `storeLastUpdate` the same type for its params.